pymarketstore/results.py

```diff
@@ -111,6 +111,26 @@ def __init__(self, reply):
     def timezone(self):
         return self.reply['timezone']
 
+    def df(self):
+        """
+        If there's a single result from this query, return its DataFrame.
+
+        If the results of this query are one row per symbol, then collapse them
+        into a single DataFrame keyed by symbol with columns of the queried
+        attribute groups (eg OHLCV -> (Open, High, Low, Close, Volume))
```

Comment:
Hmm that's interesting. While I see it's useful, I generally prefer to avoid confusion due to other factor such as data response than user's intention. Can we have a separate method name for the second case? I think I also made it slightly inconvenient by not having `df()` at the top level for the same reason, but I can accept the single result `df()` case if others feel fine.